.github/workflows/ci.yml

```
@@ -11,46 +11,19 @@ env:
-        with:
-          name: explorer-dist
-          path: crates/katana/explorer/ui/dist
-
   build:
     runs-on: ubuntu-latest-4-cores
-    needs: [fmt, cairofmt, build-explorer]
+    needs: [fmt, cairofmt]
     container:
       image: ghcr.io/dojoengine/dojo-dev:v1.2.2
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)